matchzoo/layers/DynamicMaxPooling.py

```diff
@@ -20,6 +20,17 @@ def build(self, input_shape):
     def call(self, data):
         x, self.dpool_index = data
         x_expand = K.tf.gather_nd(x, self.dpool_index)
+        stride1 = self.msize1 / self.psize1
+        stride2 = self.msize2 / self.psize2
+        
+        suggestion1 = self.msize1 / stride1
+        suggestion2 = self.msize2 / stride2
+
+        if suggestion1!=self.psize1 or suggestion2!=self.psize2:
+            print("DynamicMaxPooling Layer can not generate (%s x %s) output feature map,"
```

Comment:
Syntax level, we do not use `%s`, `print ('DynamicMaxPooling layer can not generate ({} x {}) ....'.format(variable_a, variable_b)) ` is a better choice.